src/streamlink_cli/utils/http_server.py

```diff
@@ -33,9 +33,9 @@ def addresses(self):
 
         addrs = set()
         try:
-            for info in socket.getaddrinfo(socket.gethostname(), self.port,
-                                           socket.AF_INET):
-                addrs.add(info[4][0])
+            addrs = {info[4][0] 
+                     for info in socket.getaddrinfo(socket.gethostname(), self.port, 
+                                                    socket.AF_INET)}
```

Comment:
You're redefining the set from the line above and there's another set.add() call down below. Efficiency-wise, there won't be any difference here, and regarding conciseness, please don't wrap this over multiple lines, at least not the parameters of the `socket.getaddrinfo` call, which was already unnecessary.

Reply:
Thanks for your comments, I change the format, andyes there is another set.add(), but it should improve the performance of 36-38 line and it should not affect another set.add() call down below.